MarketDataMessage2.capnp

```diff
@@ -0,0 +1,108 @@
+using Cxx = import "/capnp/c++.capnp";
+using import "Exchanges.capnp".Exchange;
+
+$Cxx.namespace("proto::marketdata");
+@0xb58df63b06ce5a55;
+
+
+#######################################################################################################
+#                   COMMON TYPES
+#######################################################################################################
+
+
+struct Header {
+    clientID @0 :UInt64;
+    senderCompID @1 :Text = "<UNDEFINED>";
+}
+
+
+enum Chanel {
+	ticker @0;          # ticker data
+    orderbook @1;       # level 2 orderbook data
```

Comment:
Can we rename to l2orderbook? I'm assuming we'll include l1orderbook and l3orderbook as options in the future.

Reply:
I think we can keep orderbook for a while. To get level 1 client can request marketDepth to 1. Level 3 is not supported by majority of exchanges.